waspc/data/Generator/templates/react-app/src/storage.ts

```diff
@@ -0,0 +1,44 @@
+export type DataStore = {
+    set(key: string, value: unknown): void;
+    get(key: string): unknown | undefined;
```

Comment:
You don't need the `| undefined` part because `unknown` is a set of all possible values: `unknown | undefined` is the same thing as `unknown`.